server/routes/auth.ts

```diff
@@ -43,7 +43,7 @@ authRoutes.post('/plex', async (req, res, next) => {
     let user = await userRepository
       .createQueryBuilder('user')
       .where('user.plexId = :id', { id: account.id })
-      .orWhere('LOWER(user.email) = :email', {
+      .orWhere('user.email = :email', {
```

Comment:
I actually wonder if this will work here. Since this is a direct query on the database I don't know if the transformer will be applied.

Reply:
@sct It does, just verified!